Documentation/api_rfc/v2_acl_api.md

```diff
@@ -0,0 +1,253 @@
+# v2 Auth and ACL
+
+## etcd Resources 
+There are two types of resources in etcd
+1. user resources: users and roles in the user store
+2. key-value resources: key-value pairs in the key-value store
+3. setting resources: acl setting, auth setting, and dynamic etcd cluster settings (election/heartbeat)
+
+### User
+A user is an identity to be authenticated. Each user can have multiple roles. The user have a capacity on the resource if one of the roles has that capacity.
+
+The special static `root` user has a ROOT role. (Caps for visual aid throughout)
+
+### Role
+Each role has exact one associated ACL. An access control list specifies the capacity on key-value resources. A role with manage capacity of a key-value resource can grant/revoke capacity of that key-value to other roles.
+
+The special static ROOT role has a full manage capacity on all key-value resources, the capacity to manage user resources and setting resources. Only the ROOT role has the capacity to manage user resources.
```

Comment:
`Only the ROOT role has the capacity to manage user resources` and setting resource?